unity-environment/Assets/ML-Agents/Scripts/Brain.cs

```diff
@@ -211,34 +211,31 @@ public void InitializeBrain(Academy aca, Communicator communicator)
         isInitialized = true;
     }
 
-    public void SendState(Agent agent, AgentInfo info)
+    public void Request(AgentInfo info, AgentAction action)
```

Comment:
What are the best names for `AgentInfo` and `AgentAction`? Maybe "predicates" and "decision"?

Reply:
We need to define the names, `Predicates` works for me (or `Features`). `Decision` unfortunately is taken by the `Decision.cs` script. `Decision`/`Action` is too restraining (only for RL), `Output` is way too general. Taking in Suggestions...